tests/src/rules/order.js

```diff
@@ -2281,7 +2325,7 @@ context('TypeScript', function () {
               parser,
               options: [
                 {
-                  groups: ['external', 'index'],
+                  groups: ['external', 'index', 'type'],
```

Comment:
same here?

Reply:
yes, I reverted the test changes from the previous commit and added new ones for the type group